src/components/PicklistSortMenu.vue

```diff
@@ -0,0 +1,92 @@
+<template>
```

Comment:
As this option is only used on a single place, so instead of creating it a component we can simply added the sort options on detail page itself.